toqito/state_props/is_abs_ppt.py

```diff
@@ -78,7 +74,10 @@ def _abs_ppt_constraints(eigvals: np.ndarray, dim: list[int]) -> list[np.ndarray
 def is_abs_ppt(mat: np.ndarray, dim: None | int | list[int] = None, tol: float = 1e-8) -> bool | int:
```

Comment:
If we are subsequently passing `tol` as `atol`, we likely should name this accordingly.